in-market-test/helper-nodes/setup_instructions.md

```diff
@@ -6,14 +6,41 @@ Here are the instructions for setting up node for running IPA.
    You can do this by running the command on the terminal: 
    `git clone https://github.com/private-attribution/ipa`
 3. This will create a folder named “ipa”. Go inside the folder.
+   
    Command: `cd ipa`
 4. Build the Rust code 
+   
    Command: `cargo build`
 5. Running a sample test : 
+   
    Command: `cargo bench --bench oneshot_ipa --features="enable-benches" --no-default-features`
    
-## Step 2: Run sample test across 3 local nodes
-   IPA team is still working on this and we will populate this section once the instructions are ready
+## Step 2: Setup helper configuration
+1. Provide details of how the machine can be accessed from public internet. For this, we would need to know
+   i. IP address/DNS which is accessible from public internet (i.e. firewall is not running)
+   ii. Confirm if machine is able to listen on port 80 
+
+2. Download docker image of IPA from shared folder. 
+    
+   We will be uploading an executable docker image which will have IPA code.
+   
+   Link: TBD
+
+3. Generate public and private keys for encrypting match keys and also, to secure the report files while in transit.
```

Comment:
would it be better if docker image already have them?

Reply:
Havent added the instructions for this, will rely on you or @bmcase to add it.